style.css

```diff
@@ -0,0 +1,126 @@
+body {
+  height: fit-content;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.general {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 375px;
+  justify-content: center;
+}
+
+header {
+  color: white;
+  background-color: #3c3a39;
+  max-width: 375px;
+  display: flex;
+  flex-direction: column;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  background-color: #1c1a19;
+  color: #fff;
+  width: 100%;
+  height: 590px;
+  justify-content: center;
+}
+
+section > * {
+  margin: 1rem;
+}
+
+section h1 {
+  color: #ff6b00;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  padding-bottom: 0;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  margin-bottom: 0;
+}
+
+section h4 {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  height: 27px;
+  color: #fff5e1;
+  margin-top: 10px;
+}
+
+section p {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin-top: 35px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
```

Comment:
- [ ] Kindly try to reduce the spacing like the previous reviewer asked you, I suggest reducing the **margin-top**
![ScreenShot_20221027235719](https://user-images.githubusercontent.com/89518488/198397531-81bce9ce-df1a-4683-adc1-831019c5220f.png).